customScripts/expansionsettings.gd

```diff
@@ -170,10 +252,11 @@ var racialstatbonuses = true
 
 #---Genealogy
 var genealogy_equalizer = 10
-var randompurebreedchance = 20
-var randommixedbreedchance = 40
+var randompurebreedchance = 10
```

Comment:
var randompurebreedchance = 20
var randommixedbreedchance = 40
var secondaryuncommonracialchance = 15
these values were changed directly instead of as tweaks. was this intended?

Reply:
Fixed